packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -218,25 +218,50 @@ export interface EcsOptimizedAmiProps {
   /**
    * What generation of Amazon Linux to use
    *
-   * @default AmazonLinux
+   * @default is Amazon Linux
```

Comment:
Also not entirely true, if you're selecting HWtype then the default is AL2.

Which I love as a behavior! But I'd also like to see it documented :)

Reply:
OK have updated default values now